pkg/apis/sparkoperator.k8s.io/v1beta1/types.go

```diff
@@ -365,6 +365,9 @@ type SparkPodSpec struct {
 	// Secrets carries information of secrets to add to the pod.
 	// Optional.
 	Secrets []SecretInfo `json:"secrets,omitempty"`
+	// Env carries the environment variables to add to the pod.
+	// Optional.
+	Env []apiv1.EnvVar `json:"env,omitempty"`
 	// EnvVars carries the environment variables to add to the pod.
 	// Optional.
 	EnvVars map[string]string `json:"envVars,omitempty"`
```

Comment:
We probably should mark this field `Deprecated` and deprecate it in the next API version.

Reply:
Done